client/new-stack-editor/lib/index.coffee

```diff
@@ -36,27 +43,25 @@ module.exports = class StackEditorAppController extends AppController
     @stackEditor.on 'InitializeRequested', @bound 'initializeStack'
 
 
-  openEditor: (templateId) ->
+  openEditor: (templateId, reset = no) ->
```

Comment:
Can we change the 2nd argument to be an object, that way, we will have to explicitly write `{ reset: no }`. IMO it improves its readability, and removes the need for checking the implementation to understand what `no` or `yes` is used for at 2nd arg.